<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkListConfigurationWidget</class>
 <widget class="QWidget" name="WorkListConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>603</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(208, 208, 208);
background-color: rgb(83, 83, 83);</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="worklistConnectionGroupBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Worklist Connection</string>
       </property>
       <widget class="QLineEdit" name="hostNameLineEdit">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>41</y>
          <width>150</width>
          <height>28</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="hostIPLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>76</y>
          <width>58</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Host IP</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="hostIPLineEdit">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>76</y>
          <width>150</width>
          <height>28</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="hostPortLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>111</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Host Port</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="hostPortSpinBox">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>111</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="localAETitleLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>149</y>
          <width>103</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Local AE Title</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="localAETitleLineEdit">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>149</y>
          <width>150</width>
          <height>28</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="remoteAETitleLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>184</y>
          <width>124</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Remote AE Title</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="remoteAETitleLineEdit">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>184</y>
          <width>150</width>
          <height>28</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="hostNameLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>41</y>
          <width>90</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Host Name</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="worklistGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Worklist Field Configuration</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="patientIDCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Patient Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="accessionNumberCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Accession Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="otherPatientIDCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Other Patient Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="admissionIDCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Admission Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="scheduledProcedureStepIDCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Scheduled Procedure Step Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="requestedProcedureIDCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Requested Procedure ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="requestedProcedureCodeValueCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Requested Procedure Code Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="studyInstanceUIDCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Study Instance UID</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="workListConfigurationGroupBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Worklist Configuration</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="modalityLbl">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Modality</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="modalityComboBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>DX</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DX/DR</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="scopeLbl">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Scope</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="scopeComboBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>This station</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All stations</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="enableWorkListCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Enable worklist</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="ignoreEchoFailCheckBox">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ignore C-ECHO failure</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="workListFieldGroupBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Worklist Field</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="fieldListView"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="groupNameLbl">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Group name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="groupNameLineEdit">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="dicomGroupLbl">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>DICOM group</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="dicomGroupLineEdit">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="dicomElementLbl">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>DICOM element</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="dicomElementLineEdit">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="parentNodeGroupLbl">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Parent node group</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="parentNodeGroupLineEdit">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="parentNodeELementLbl">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Parent node element</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="parentNodeELementLineEdit">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="updateTagBtn">
              <property name="text">
               <string>Update Tag</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testConnectionBtn">
              <property name="text">
               <string>Test Connection</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
